client/branded/src/global-styles/code.scss

```
@@ -13,7 +13,7 @@ code,
 .code {
     font-family: $code-font-family;
     font-size: $code-font-size;
-    line-height: 16px;
+    line-height: 1.25;
```

1.25*12px = 15px, not 16px? Shouldn't this be 16/12?
```suggestion
    line-height: (16/12);
```